.rubocop.yml

```diff
@@ -12,10 +12,15 @@ Lint/EmptyClass:
   Enabled: false
 Naming/PredicateName:
   Enabled: false
+Naming/MethodParameterName:
+  Exclude:
+  - lib/game.rb
```

Comment:
What was the previous error?

Reply:
Rubocop was upset that the method name "ui" was less than 3 characters long.